packages/perspective-viewer-datagrid/src/js/datagrid.js

```diff
@@ -101,27 +103,62 @@ function _tree_header(td, path, is_leaf, is_open) {
         for (let i = 0; i < path.length; i++) {
             html += `<span class="pd-tree-group"></span>`;
         }
-        td.innerHTML = `<div style="display:flex;align-items:stretch">${html}<span class="pd-group-name" style="margin-left:12px">${name}</span></div>`;
+        td.innerHTML = `<div style="display:flex;align-items:stretch">${html}<span class="pd-group-name pd-group-leaf">${name}</span></div>`;
     } else {
         const icon = is_open ? "remove" : "add";
         let html = "";
         for (let i = 0; i < path.length; i++) {
-            html += `<span style="margin-left:5px;margin-right:15px;border-left:1px solid #eee;height:19px"></span>`;
+            html += `<span class="pd-tree-group"></span>`;
         }
         td.innerHTML = `<div style="display:flex;align-items:stretch">${html}<span class="pd-row-header-icon">${icon}</span><span class="pd-group-name">${name}</span></div>`;
     }
     td.classList.add("pd-group-header");
 }
 
+async function getCellConfig(view, config, row_idx, col_idx) {

```

Comment:
This is prime for a refactor as some of the code exists in different forms in the other plugins.

Reply:
agreed